components/MenuBar/MenuBar.jsx

```diff
@@ -0,0 +1,76 @@
+import { PropTypes, Component } from 'react'
+
+require('./MenuBar.scss')
+
+class MenuBar extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {mobile: false}
+  }
+
+  componentWillMount() {
+    this.handleResize(this)
+    const handleResize = this.handleResize
+    const thisRef = this
```

Comment:
Thanks @nlitwin I was wondering how can do that and when I didn't any way that time, I used thisRef. :) I would fix it.